src/SSRv1/templates/newItem.php

```diff
@@ -30,14 +32,19 @@
 		<?php if(isset($base) && $base->hasCode()): ?>
 			<div class="inline-alert alert-info" role="alert"><i class="fa fa-info-circle"></i>️&nbsp;This is a copy of <span class="text-monospace"><?= $base->getCode() ?></span>, remember to change serial numbers, notes, working status, etc...</div>
 		<?php unset($noticeFeature); endif; ?>
+
+		<?php if(isset($importedFrom)): ?>
+			<div class="inline-alert alert-info" role="alert"><i class="fa fa-info-circle"></i>️&nbsp;This item is generated from a bulk import ( with Id = <span class="text-monospace"><?= $importedFrom ?> ) </span></div>
+			<?php unset($noticeFeature); endif; ?>
```

Comment:
```suggestion
			endif; ?>
```

Oops, sorry, `unset($noticeFeature)` was an accidental copy paste in the line above, already... Remove it from both places, it's not used in this template